ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh

```diff
@@ -23,6 +23,9 @@ current_worker_count=$(oc get nodes --no-headers -l node-role.kubernetes.io/work
 iteration_multiplier=$(($ITERATION_MULTIPLIER_ENV))
 export ITERATIONS=$(($iteration_multiplier*$current_worker_count))
 export WORKLOAD=cluster-density-v2
+echo "######"
+export UUID="{`uuidgen`-cpt}"
```

Comment:
@vishnuchalla , do we need a suffix or a prefix?

Reply:
Updated, ptal!